addons/l10n_in_withholding/models/account_move.py

```diff
@@ -82,8 +83,8 @@ def _get_sections_aggregate_sum_by_pan(self, section_alert, commercial_partner_i
             query = self.env['account.move.line']._where_calc(default_domain + frequency_domain)
             result = self.env.execute_query_dict(SQL(
                 """
-                SELECT sum(account_move_line.balance) as balance,
-                       sum(account_move_line.price_total * am.invoice_currency_rate) as price_total
+                SELECT COALESCE(sum(account_move_line.balance), 0) as balance,
+                       COALESCE(sum(account_move_line.price_total * am.invoice_currency_rate), 0) as price_total
```

Comment:
query will return ```None``` if it does not find any records, as only posted records are considered. This may break the flow. Therefore, ```COALESCE``` is used, which will set the fallback value to ```0```.